libs/appstore/src/portal/zomes/portal-zome-client.ts

```diff
@@ -128,16 +129,16 @@ export class PortalZomeClient extends ZomeClient {
     statusCallback = () => {},
   }: TryWithHostsArgs<T>): Promise<T> {
     // try with first responding host
-    statusCallback('Getting available host...');
+    statusCallback('Searching available peer');
     const quickestHost: AgentPubKey = await this.getAvailableHostForZomeFunction(
       dnaZomeFunction,
       pingTimeout,
     );
 
-    console.log('got quickest host: ', quickestHost);
+    console.log('got quickest host: ', encodeHashToBase64(quickestHost));
```

Comment:
This `encodeHashToBase6` call intentional?

Reply:
Yes it's on purpose since it's easier to read in the console and compare with expected agent keys.